packages/block-editor/src/components/writing-flow/use-input.js

```diff
@@ -66,7 +69,24 @@ export default function useInput() {
 				node.contentEditable = false;
 				event.preventDefault();
 				if ( __unstableIsFullySelected() ) {
-					removeBlocks( getSelectedBlockClientIds() );
+					const shouldDisplayRemovalPrompt =
```

Comment:
Maybe this should be something like `blockNamesWithWarnings` or something way better. `should*` makes me thing the return values is a bool